nixos/tests/etcd/etcd.nix

```diff
@@ -1,25 +1,34 @@
 # This test runs simple etcd node
-
 import ../make-test-python.nix ({ pkgs, ... } : {
   name = "etcd";
-
   meta = with pkgs.lib.maintainers; {
     maintainers = [ offline ];
   };
 
   nodes = {
     node = { ... }: {
-      services.etcd.enable = true;
+      services.etcd = {
+        enable = true;
+        # Ensure etcd is ready to accept connections
+        extraConf = {
+          "initial-advertise-peer-urls" = "http://localhost:2380";
```

Comment:
This should be a default value for the service configuration. We should assume it works locally until not.

Reply:
ok

Reply:
#352599